src/lock-bot.ts

```diff
@@ -25,6 +25,7 @@ export type Destination = "user" | "channel";
 export interface Response {
```

Comment:
If you scroll up a bit you can see the `LockRepo` interface.

I think we'll have to return an additional `description?: string` from `getAll` so that the description can be seen in `/locks`. There will also need to be some additional tests and implementation etc. such has displaying locks with and without a description.

Then for `setOwner` I think it would make sense to pass `description?: string` after `team` and before `metadata`

![Screen Shot 2022-02-13 at 8 39 28 PM](https://user-images.githubusercontent.com/10026538/153777312-111516b5-4c79-439d-91bd-074b85c38c83.png)